README.md

```diff
@@ -47,7 +47,7 @@ For example, any given Jaeger installation at Uber is typically processing sever
 ### Native support for OpenTracing
 
 Jaeger backend, Web UI, and instrumentation libraries have been designed from the ground up to support the [OpenTracing standard](https://opentracing.io/specification/).
-  * Represent traces as directed acyclic graphs (not just trees) via [span references](https://github.com/opentracing/specification/blob/master/specification.md#references-between-spans)
+  * Represent traces as directed acyclic graphs (not just trees) via [span references](https://github.com/opentracing/specification/blob/main/specification.md#references-between-spans)
```

Comment:
this 404s for me

Reply:
Good catch, this shouldn't have been renamed.